docs/toolbox/accessibility_indicators/catchments.md

```diff
@@ -25,19 +25,19 @@ You might know this feature from our previous software versions under the terms
 :::
 
 :::info 
-The calculation of Catchment Areas is only possible in areas where the transport network of the selected routing mode is integrated in GOAT. As soon as you selected a routing mode, you will see the **geofence** in which the calculation is possible.
+The calculation of the catchment areas is only available for areas where the transport network of the selected routing mode is integrated into GOAT. As soon as you select a routing mode, you will see the **geofence** in which the calculation is possible.
 
 <div style={{ display: 'flex', flexDirection: 'column', alignItems: 'center' }}>
-  <img src={require('/img/toolbox/accessibility_indicators/catchments/geofence.png').default} alt="Geofence for catchment area calculation in GOAT" style={{ maxHeight: "400px", maxWidth: "400px", alignItems:'center'}}/>
+  <img src={require('/img/toolbox/accessibility_indicators/gueteklassen/geofence-pt.png').default} alt="Geofence for catchment area calculation in GOAT" style={{ maxHeight: "400px", maxWidth: "400px", alignItems:'center'}}/>
```

Comment:
Walk, bicycle, pedelec and car catchment areas can be calculated anywhere in Europe, so we can keep the previous geofence. GOAT will dynamically show a more limited geofence if the user selects PT.